fastlane/lib/fastlane/actions/git_tag_exists.rb

```diff
@@ -2,8 +2,19 @@ module Fastlane
   module Actions
     class GitTagExistsAction < Action
       def self.run(params)
-        result = Actions.sh("git rev-parse -q --verify refs/tags/#{params[:tag].shellescape} || true", log: FastlaneCore::Globals.verbose?).chomp
-        !result.empty?
+        tag_ref = "refs/tags/#{params[:tag].shellescape}"
+        if params[:remote]
```

Comment:
wouldn't it be simpler to just use the existance of `remote_name` to decide if a remote should be checked?

Reply:
Good point!

Reply:
Hmm, but in that case we cannot provide default `remote_name` value..
It's fine to me but most of other `git remote` related actions have default value for `remote`.

Reply:
Also true, right.
What other actions are there? Quick google only found me https://docs.fastlane.tools/actions/push_git_tags/

Reply:
There is push_to_git_remote I found.
https://github.com/fastlane/fastlane/blob/master/fastlane/lib/fastlane/actions/push_to_git_remote.rb

Reply:
Hmm... problem in comparing those is that both `push_git_tags` and `push_to_git` by default interact with a remote, so having a `remote` param with default value `origin` makes total sense. But this one works locally by default, and only with this change can does remote things, right?

Reply:
(Which of course doesn't help us solve the problem at all :p)

Reply:
Correct, and it totally makes sense. Had a great discussion here, thanks! 
Will fix it only to use `remote_name`!

Reply:
That wasn't really my conclusion though. I think you are good with how it is right now if you want to.